engine/clientdb/queries.sql

```diff
@@ -40,15 +40,18 @@ INSERT INTO logs (
     ?, ?, ?, ?, ?, ?, ?, ?, ?, ?
 ) RETURNING id;
 
--- -- name: InsertMetric :one
--- INSERT INTO metrics (
---     name, description, unit, type, timestamp, data_points
--- ) VALUES (
---     ?, ?, ?, ?, ?, ?
--- ) RETURNING id;
+-- name: InsertMetric :one
+INSERT INTO metrics (
+    data
```

Comment:
That works - let's see how it goes. I would leave the rest as-is for now, since we've tossed around ideas for exposing OTel data through the Dagger API somehow, and it'll be easier to do things like "subscribe to this span" with a proper schema.